WordPress/twentytwentyfive
# Add Roboto Slab and Beiruti

**Description**
Adds two missing fonts: Roboto Slab and Beiruti.

Partial for https://github.com/WordPress/twentytwentyfive/issues/22

**Testing Instructions**

1. Apply the PR
2. Add a block that has font family options.
3 Confirm that the fonts Roboto Slab and Beiruti can be selected in the font family option
5. Confirm if the fonts show correctly in the editors and front.
